File: trunk/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/ReverseOrgDocNotExistException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class ReverseOrgDocNotExistException : Exception
    {
        public ReverseOrgDocNotExistException()
            : base("Document to be reversed does not exist")
        {
        }
    }
}

File: branches/V1.0/proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/ReverseOrgDocNotExistException.java
package com.jasonzqshen.familyaccounting.core.exception;

public class ReverseOrgDocNotExistException extends Exception {
	/**
	 * 
	 */
	private static final long serialVersionUID = -4377933257679741211L;

	public ReverseOrgDocNotExistException() {
		super("Document to be reversed does not exist");
	}
}
